src/main/java/org/openhab/automation/jrule/internal/engine/excutioncontext/JRuleLocalTimerExecutionContext.java

```diff
@@ -23,10 +23,13 @@
  */
 public class JRuleLocalTimerExecutionContext extends JRuleExecutionContext {
 
+    private JRuleExecutionContext parentContext;
     private String timerName;
 
     public JRuleLocalTimerExecutionContext(JRuleExecutionContext parentContext, String timerName) {
-        super(parentContext.getRule(), parentContext.getLogName(), null, null, null);
+        super(parentContext.getRule(), parentContext.getLogName(), parentContext.getLoggingTags(),
+                parentContext.getMethod(), parentContext.getPreconditionContextList());
```

Comment:
Will inheriting the parents preconditions drop execution of a local timer if the preconditions of the source rule trigger changes?

Reply:
Effectively there are no preconditions for a timer. Its just for completness.